app/views/users/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1>User Form</h1>
+
+<%#
+  <form action="/users" method="POST" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %> <%# ">
+  <label for="username"> Username </label>
+  <input type="text" id="username" name="user[username]">
+  <label for="email"> Email </label>
+  <input type="email" id="email" name="user[email]">
+  <label for="password"> Password </label>
+  <input type="password" id="password" name="user[password]">
+  <input type="submit" value="Submit">
+</form>
+%>
+
+<%# form_tag %>
+
+<%
+=begin %>
+<%= form_tag users_path do %>
+  <%= label_tag :username %>
+  <%= text_field_tag :username, params[:username] %>
+  <%= label_tag :email %>
+  <%= text_field_tag :email, params[:email] %>
+  <%= label_tag :password %>
+  <%= password_field_tag :password, params[:password] %>
+  <%= submit_tag "Submit" %>
+<% end %>
+<% 
+=end %>
+
+<%# form_for %>
```

Comment:
- [ ] [MANDATORY] Your form_for implementation is commented out as requested but it still needs to be completed as below:
```
<%#= form_for @user do |u|%> 
<%#= u.label :username %> <%= u.text_field :username%> 
...
...
<%#= u.submit %> 
<%# end %>
```
NB: Remember to fill out all the fields.